applications/directory/src/components/WorkExamples.js

```diff
@@ -6,16 +7,36 @@ function WorkExamples({ images }) {
   return (
     <Card
       sx={{
-        '& h2': { typography: 'body1', fontSize: '20px', fontWeight: '500' }
+        marginBottom: '3.125rem',
+        boxShadow: '-0.625rem 0.625rem 1.25rem 0 rgba(30, 30, 30, 0.05)'
       }}
     >
-      <h2>Work Examples</h2>
-      <Box sx={{ display: 'grid', gridTemplateColumns: 'repeat(3, 1fr)' }}>
+      <Typography
+        variant="h2"
+        sx={{
+          fontSize: '1.25rem',
+          fontFamily: 'gira-sans, sans-serif',
+          fontWeight: 500,
+          marginBottom: '0.875rem'
+        }}
+      >
+        Work Examples
+      </Typography>
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: [
```

Comment:
Nice job using Grid 👍

Reply:
I think Jacky started it :) I did not know you like grid